project_todo.md

```diff
@@ -4,6 +4,4 @@
 1. Add unit tests for every entry point in `pandas_text`
 1. Implement the remainder of the Pandas `ExtensionArray` API
 1. Add tests that use Pandas' built-in test framework for extension arrays
-1. Prioritize remaining Gremlin functionality
-1. Refactor `gremlin.py` into a sub-module
 1. Rename `pandas_text` to `text_pandas` for better Naming compliance
```

Comment:
Actually, I think that this entire file can be removed. #1 has a more up-to-date version of the same information.